Source/Core/getStringFromTypedArray.js

```diff
@@ -1,11 +1,13 @@
 define([
         './defaultValue',
         './defined',
-        './DeveloperError'
+        './DeveloperError',
+        './RuntimeError'
     ], function(
         defaultValue,
         defined,
-        DeveloperError) {
+        DeveloperError,
+        RuntimeError) {
     'use strict';
     /*global TextDecoder*/
```

Comment:
I know it has nothing to do with this PR, but this line can be removed, I assume it's a leftover from jsHint.